jsk_topic_tools/src/lightweight_throttle_nodelet.cpp

```diff
@@ -94,13 +105,21 @@ namespace jsk_topic_tools
       advertised_ = true;
       pub_ = pnh_.advertise(opts);
     }
+
     ros::Time now = ros::Time::now();
-    if ((now - latest_stamp_).toSec() > 1 / update_rate_) {
+
+    if (latest_stamp_ > now) {
+      ROS_WARN("Detected jump back in time.");
```

Comment:
Because we can clearly recognize that time is jumped back and I think there is no one who want to know the fact that 'Overwriting latest_stamp', because most of users should not be conscious about `latest_stamp_` variable and core developers can know immediately this simple mechanism.